app/controllers/admin/items/tickets_controller.rb

```diff
@@ -47,6 +47,11 @@ def destroy
 
       private
 
+      def update_item_status!(ticket)
+        status = ticket.retired? ? Item.statuses["retired"] : Item.statuses["maintenance"]
+        @ticket.item.update!(status:)
+      end
```

Comment:
I wonder if there is a workflow where staff would want to create a ticket without putting an item into maintenance? Let's merge this as-is but check in at the next stakeholders about this.